packages/validator-ajv8/src/validator.ts

```diff
@@ -202,14 +203,45 @@ export default class AJV8Validator<
     errors: ErrorObject[] = []
   ): RJSFValidationError[] {
     return errors.map((e: ErrorObject) => {
-      const { instancePath, keyword, message, params, schemaPath } = e;
+      const {
+        instancePath,
+        keyword,
+        params,
+        schemaPath,
+        parentSchema,
+        ...rest
+      } = e;
+      let { message } = rest;
+
       let property = instancePath.replace(/\//g, ".");
-      let stack = `${property} ${message}`.trim();
+
+      let stack: string | undefined = undefined;
+
       if ("missingProperty" in params) {
         property = property
           ? `${property}.${params.missingProperty}`
           : params.missingProperty;
+
+        const currentProperty = property.split(".").slice(-1)[0];
```

Comment:
Couldn't you just use `params.missingProperty` instead of all this jiggling since it will always be the last one?

Reply:
@heath-freenome in the case of a nested property, property would be like this `.nested.lastName` so I do think I need to get the last item. 

I could possibly use a regex if you think that is better?

Reply:
Great shout, that is much better.